staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go

```diff
@@ -427,6 +427,12 @@ type PersistentVolumeBinderControllerConfiguration struct {
 	PVClaimBinderSyncPeriod metav1.Duration
 	// volumeConfiguration holds configuration for volume related features.
 	VolumeConfiguration VolumeConfiguration
+	// DialHostCIDRDenylist is a list of CIDRs that should not be dialed by the
+	// controller from plugins.
+	DialHostCIDRDenylist *[]string
+	// DialHostDenyNonproxyable indicates if nonproxyable hosts should be denied
+	// when dialing from plugins.
+	DialHostDenyNonproxyable bool
```

Comment:
also, is there a better name than `DialHostDenyNonproxyable`? We don't proxy to volume plugin hostnames, so it's a bit odd to make the config/flag call out "nonproxyable" addresses. Is that a term of art that users will know means loopback + link-local-unicast + link-local-multicast + interface-local-multicast?

cc @thockin in case there's a nice standard term for this set of IP ranges.

might be easier to understand what the option is doing if it is opt in and defaults false (in 1.19+), something like:
```
VolumeHostAllowLocalLoopback *bool
```